dali/pipeline/operators/reader/loader/loader.cc

```diff
@@ -19,9 +19,11 @@ namespace dali {
 
 DALI_SCHEMA(LoaderBase)
   .AddOptionalArg("random_shuffle",
-      R"code(Whether to randomly shuffle data.)code", false)
+      R"code(Whether to randomly shuffle data. Prefetch buffer of `initial_fill` size is used
+to sequentially read data and then randomly sample it to form a batch.)code", false)
   .AddOptionalArg("initial_fill",
-      R"code(Size of the buffer used for shuffling.)code", 1024)
+      R"code(Size of the buffer used for shuffling. If `random_shuffle` is off then
```

Comment:
Maybe « If `random_shuffle` is off, this parameter is ignored. »?

Reply:
Done